packages/opentelemetry-sdk-trace-web/test/utils.test.ts

```diff
@@ -213,8 +213,6 @@ describe('utils', () => {
       const time = createHrTime(startTime, 500);
       sinon.stub(performance, 'timeOrigin').value(0);
       sinon.stub(performance, 'now').callsFake(() => hrTimeToNanoseconds(time));
-
-      sinon.stub(core, 'hrTime').returns(time);
```

Comment:
This is not a configurable property, but it turns out not to be required to make the tests pass anyway.